docs/fallbacks.md

```diff
@@ -1,44 +1,112 @@
 # Fallbacks
 
-### Circuit Breaker
+## Circuit Breaker
 
 A simple implementation of [Circuit Breaker](http://martinfowler.com/bliki/CircuitBreaker.html) pattern.
```

Comment:
What about removing "simple" from this phrase? We can let the reader decide whether it is simple or not.